app/src/main/java/com/hamdan/forzenbook/view/login/CreateAccount.kt

```diff
@@ -215,19 +265,27 @@ private fun Content(
             disabledLeadingIconColor = MaterialTheme.colors.primary,
             disabledTrailingIconColor = MaterialTheme.colors.secondary
         ),
-        singleLine = true
+        textStyle = TextStyle(fontSize = ForzenbookTheme.typography.h2.fontSize),
+        singleLine = true,
```

Comment:
You have maxlines = 1 above and singleLine here. Pick one

Reply:
I swear I still saw maxLines above. You left the singleLine and resolved the conversation without changing it? <_<

Reply:
I thought I changed it to max lines, idk why it isn't. changed it to max lines now.